src/components/atoms/inputTextArea/inputTextArea.scss

```
@@ -3,4 +3,15 @@ textarea {
   box-shadow: $input-shadow;
   border-radius: $regular-radius;
   min-height: 150px;
-}
+
+  &:hover {
+    background-color: $primary-utility-color;
+    border: 1px solid $primary-dark-color;
+    border-radius: $regular-radius;
+  }
+
+  &:focus {
+    border: 2px solid $gray-lighter;
+    border-radius: $regular-radius;
+  }
+}
```

```suggestion
}

```
manque un retour a la ligne